Package.swift

```diff
@@ -28,13 +28,13 @@ let package = Package(
 #if canImport(ExampleConfig)
 import ExampleConfig
 
-ExampleConfig(value: "example value").write()
+let example = ExampleConfig(value: "example value").write()
```

Comment:
So, is this a necessary change for everyone writing a `Package.swift` now? I'll mention it in the CHANGELOG, but is there any way to detect the lack of it?

Reply:
Expressions at top level are not allowed anymore. 🤷‍♂️

Reply:
Sure, swift can do swift I guess